toolbelt/sockets.cc

```diff
@@ -455,10 +454,10 @@ TCPSocket::TCPSocket() : NetworkSocket(socket(AF_INET, SOCK_STREAM, 0)) {}
 
 absl::Status TCPSocket::Bind(const InetAddress &addr, bool listen) {
   bool binding_to_zero = addr.Port() == 0;
-  int e =
+  int bind_err =
```

Comment:
What's wrong with 'e'?

Reply:
The `e` appeared multiple times at different scopes, thus triggering the `-Wshadow` warnings. I guess one could remain as `e`, but I figured if I have to change one of the two, I might as well make both more explicitly named.

Reply:
OK, no problem.  I was just wondering if your warnings were complaining about single letter names or something.

I had to do a similar exercise when I imported all of this into Cruise, and a lot more stuff Axivion was complaining about.  Really pedantic.  I have changed the software quite a lot for the cruise usage and I hope to feed some of it back to here.  For subspace, coroutines and toolbelt.  I am also enhancing them for something new I'm working on at home that might interest you: MCP and syscon, but don't much better this time.

Reply:
Cool. Yeah, I got issues with the short names too, but that was too much to migrate, so I've disabled that check for now. But I guess this change fixes that too.

On the MCP + syscon... interesting... but I have rolled my own already, and it kicks butt. ;)

Reply:
I thought you might have been.  Do you have zygotes and modules?  I have those, all using coroutines so there isn't a thread in sight.  I will be releasing mine on github soon, once I get the documentation done and port it from MacOS to Linux (easy).

Reply:
I made a modules system much superior to IPC modules. An MCP equivalent that combines sysconfig-like thing, but overall much lighter-weight. And a clock server that uses your coroutines.

No zygotes forking or daemon (MARS) sort of system though, since I don't think we'll need that.